native/cocos/renderer/gfx-base/GFXDef-common.h

```diff
@@ -1278,6 +1288,9 @@ struct ALIGNAS(8) ColorAttachment {
 #if CC_CPU_ARCH == CC_CPU_ARCH_64
     uint32_t _padding{0};
 #endif
+    void *externalResLow{nullptr}; // CVPixelBuffer for Metal, EGLImage/Texture2D for GLES
```

Comment:
ColorAttachment is a layout data structure. Should not be associated with a specific instance.

Reply:
> ColorAttachment is a layout data structure. Should not be associated with a specific instance.

ok, new commit has pushed